packages/webview/README.md

```diff
@@ -0,0 +1,35 @@
+# webview-background
+
+A small plugin to change the background of a webview.
+
+## Install
+
+```bash
+npm install webview-background
```

Comment:
Yes, it would be preferable to use the same package manager throughout the project.

Originally suggested here: https://github.com/cybersemics/em/pull/2594#discussion_r1864315059